flight/Modules/Stabilization/stabilization.c

```diff
@@ -813,6 +917,18 @@ static void stabilizationTask(void* parameters)
 
 		uint16_t max_safe_rate = PIOS_SENSORS_GetMaxGyro() * 0.9f;
 
+#if defined(PIOS_INCLUDE_LQG)
+		bool lqg_in_use = false;
+
+		for (int i = 0; i < MAX_AXES; i++) {
```

Comment:
So this basically means, if you enter into LQG in flight, it's going to briefly be flying acro instead?  Do we have an idea of what the bounds of this is, in flight, like on a F3?

It's a shame it's not easy to tell if it's in any flight mode or active stab bank.... sanitycheck knows, we don't.

Reply:
If there's values in SystemIdent, it starts to solve right after boot up. It does a set amount of cycles for RTKF (150) and LQR (1500), then it stops doing the math stuff and immediately returns.

When I moved the solver stuff into stab.c, I initially forgot to initialize the LQGSolution UAVO. So when it tried to store the solution data, it crashed. Took about a second to crash after hitting Boot in bootloader mode. Long story short, it has the LQR gains way before someone arms the quad. Would probably apply to F3, too.

Reply:
Ah, my bad.  Great.